pyro/poutine/block_poutine.py

```diff
@@ -1,9 +1,35 @@
 from __future__ import absolute_import, division, print_function
 
-from .poutine import Messenger, Poutine
+from .poutine import Messenger
 
 
 class BlockMessenger(Messenger):
+    # TODO fix this docstring
```

Comment:
Is this TODO meant for your follow-up PR? If so then LGTM.

Reply:
Yes